invenio/values.yaml

```diff
@@ -49,6 +32,11 @@ invenio:
       - name: ""
         consumer_key: ""
         consumer_secret: ""
+  extra_config:
+    INVENIO_COLLECT_STORAGE: "flask_collect.storage.file"
+    INVENIO_DATACITE_ENABLED: "False"
```

Comment:
The only hassle is that due to how values are injected in the config, it expects a string so numbers and booleans need to be wrapped around with quotes. The result configMap is correct, i.e. the app understands correctly bools and numbers. (tested in zenodo-rdm-qa).